app/models/organization/FreeCreditTransactionService.scala

```diff
@@ -0,0 +1,40 @@
+package models.organization
+
+import com.scalableminds.util.tools.{Fox, FoxImplicits}
+import com.scalableminds.webknossos.datastore.helpers.IntervalScheduler
+import com.typesafe.scalalogging.LazyLogging
+import controllers.Controller
+import org.apache.pekko.actor.ActorSystem
+import play.api.inject.ApplicationLifecycle
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+
+class FreeCreditTransactionService @Inject()(creditTransactionDAO: CreditTransactionDAO,
+                                             val lifecycle: ApplicationLifecycle,
+                                             val system: ActorSystem)(implicit val ec: ExecutionContext)
+    extends Controller
+    with FoxImplicits
+    with LazyLogging
+    with IntervalScheduler {
+
+  override protected def actorSystem: ActorSystem = system
```

Comment:
This line can be removed if the injected `val system: ActorSystem` is renamed to `val actorSystem: ActorSystem`.